internals/overlord/logstate/manager.go

```diff
@@ -44,5 +116,13 @@ func (m *LogManager) Ensure() error {
 
 // Stop implements overlord.StateStopper and stops all log forwarding.
 func (m *LogManager) Stop() {
-	// TODO: implement
+	wg := sync.WaitGroup{}
+	for _, f := range m.forwarders {
+		wg.Add(1)
+		go func(f *logForwarder) {
```

Comment:
why do we need the goroutine here to stop the forwarder? the forwarder stop just closes a channel.